mappings/net/minecraft/block/entity/EndGatewayBlockEntity.mapping

```diff
@@ -42,17 +43,23 @@ CLASS net/minecraft/class_2643 net/minecraft/block/entity/EndGatewayBlockEntity
 	METHOD method_11421 needsCooldownBeforeTeleporting ()Z
 	METHOD method_30276 canTeleport (Lnet/minecraft/class_1297;)Z
 		ARG 0 entity
-	METHOD method_31698 (Lnet/minecraft/class_3218;Lnet/minecraft/class_243;)Z
+	METHOD method_31698 isChunkEmptyAt (Lnet/minecraft/class_3218;Lnet/minecraft/class_243;)Z
 		ARG 0 world
-	METHOD method_31699 (Lnet/minecraft/class_3218;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2338;
+		ARG 1 pos
+	METHOD method_31699 setupExitPortalLocation (Lnet/minecraft/class_3218;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2338;
+		COMMENT Finds teleport location and creates an island to teleport to (if there is none).
```

Comment:
```suggestion
	METHOD method_31699 setupExitPortalPosition(Lnet/minecraft/class_3218;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2338;
		COMMENT Finds teleport position and creates an island to teleport to (if there is none).
```

Reply:
This suggestion removes a space between mapping and descriptor and breaks the enigma format.